### ISSUE TITLE
Schema compare select source/target: Database selection always select first database in list when editor accessibility support is on

### ISSUE THREAD

**Issue by leonardder** [microsoft/azuredatastudio#13499]
Issue Type: <b>Bug</b> 

1. Ensure Editor Accessibility support is enabled in Data Studio settings
2. Open a connection
3. Open a Database project
4. Right click > Schema compare.
5. Choose select target
6. Select database as target
7. Select the sql server for the just opened connection
8. Select the desired database
9. Press Ok

Expected: The selected database is chosen as target
Actual: The first database in the databases list is chosen as target

Azure Data Studio version: azuredatastudio 1.24.0 (eb3c0f5d9df86b9490606aab7fffaa253b0aca4d, 2020-11-10T23:38:09.705Z)
Schema Compare Version: 1.7.0
OS version: Windows_NT x64 10.0.19042

**Comment by kisantia** [comment]
I'm unable to repro this when following the steps(the selected database is selected, not always the first database in the list). Can you share of gif of this?

**Comment by leonardder** [comment]
I probably must ave mentioned that I was using the NVDA screen reader 
while reproducing this. While a colleague of mine used a mouse to do 
exactly the same on my system while NVDA was running. Let us get back to 
you after we did some additional attempts to reproduce it on another system.

**Comment by leonardder** [comment]
I did some tests and found out that the bug is related to having the     editor.accessibilitySupport setting set to on. If that setting is off, selection of the database works as expected. I will update the issue description accordingly.

**Comment by leonardder** [comment]
@isidorn: As Data Studio borrows its editor accessibility support from VS Code, do you think there's anything that can clarify this? 
It looks like editor accessibility support touches accessibility of combo boxes somehow? When accessibility support is on, the server / database combo boxes have one entry in the window's tab order and are reported as combo box. If accessibility support is off, the combo boxes have two entries in the window's tab order, namely an edit box and a button that expands the combo box. The expanded combo box is reported by screen readers as tree view.

**Comment by isidorn** [comment]
@leonardder I believe the Data Studio uses a custom web view which does not use VS Code widgets (which are accessible) and in that lyes the problem.
However if Data Studio is using some VS Code widget and it is not accessible please let me know and I can look into this from the VS Code side.

**Comment by leonardder** [comment]
@isidorn Some details in the hope they can help you

The combo box is wrapped in an element with tag modelview-dropdown

HTML when accessibility support is on:

```
<modelview-dropdown style="overflow: hidden; position: relative;">

	<div>
		<div style="width: 100%; position: relative; display: none;">
			<div class="modelview-loadingComponent-spinner" style="position:absolute; right: 0px; margin-right: 5px; height:15px; z-index:1"></div>
			<div style="width: 100%; display: none;" class=""><div class="monaco-selectbox idle"></div><select class="monaco-select-box monaco-select-box-dropdown-padding" title="Loading" aria-label="Server" disabled="" style=""><option value="Loading">Loading</option></select></div>
		</div>
		<div style="width: 100%;"><span class="monaco-dropdown-width-control-element" aria-hidden="true">SQLC05-SQL (default)</span><div class="monaco-dropdown" style="width: 100%;"><div class="dropdown-input" style="width: 100%;"><div class="monaco-inputbox idle" style="background-color: rgb(255, 255, 255); color: rgb(74, 74, 74); border-width: 1px; border-style: solid; border-color: rgb(136, 136, 136);"><div class="wrapper"><input class="input" autocorrect="off" autocapitalize="off" spellcheck="false" type="text" wrap="off" title="" aria-label="Server" style="padding-right: 22px; background-color: inherit; color: rgb(74, 74, 74);"></div><div class="monaco-action-bar animated"><ul class="actions-container" role="toolbar"><li class="action-item" role="presentation"><a class="action-label codicon dropdown-arrow" role="button" tabindex="0"></a></li></ul></div></div></div></div></div>
		<div style="width: 100%; display: none;" class="select-container"><div class="monaco-selectbox idle"></div><select class="monaco-select-box monaco-select-box-dropdown-padding" aria-label="Server" style="background-color: rgb(255, 255, 255); color: rgb(74, 74, 74); border-color: rgb(136, 136, 136);"></select></div>
	</div>
	</modelview-dropdown>
```

HTML when accessibility support is off:

```
<modelview-dropdown style="overflow: hidden; position: relative;">

	<div>
		<div style="width: 100%; position: relative; display: none;">
			<div class="modelview-loadingComponent-spinner" style="position:absolute; right: 0px; margin-right: 5px; height:15px; z-index:1"></div>
			<div style="width: 100%; display: none;" class=""><div class="monaco-selectbox idle"></div><select class="monaco-select-box monaco-select-box-dropdown-padding" title="Loading" aria-label="Server" disabled="" style=""><option value="Loading">Loading</option></select></div>
		</div>
		<div style="width: 100%;"><span class="monaco-dropdown-width-control-element" aria-hidden="true">SQLC05-SQL (default)</span><div class="monaco-dropdown" style="width: 100%;"><div class="dropdown-input" style="width: 100%;"><div class="monaco-inputbox idle" style="background-color: rgb(255, 255, 255); color: rgb(74, 74, 74); border-width: 1px; border-style: solid; border-color: rgb(136, 136, 136);"><div class="wrapper"><input class="input" autocorrect="off" autocapitalize="off" spellcheck="false" type="text" wrap="off" title="" aria-label="Server" style="padding-right: 22px; background-color: inherit; color: rgb(74, 74, 74);"></div><div class="monaco-action-bar animated"><ul class="actions-container" role="toolbar"><li class="action-item" role="presentation"><a class="action-label codicon dropdown-arrow" role="button" tabindex="0"></a></li></ul></div></div></div></div></div>
		<div style="width: 100%; display: none;" class="select-container"><div class="monaco-selectbox idle"></div><select class="monaco-select-box monaco-select-box-dropdown-padding" aria-label="Server" style="background-color: rgb(255, 255, 255); color: rgb(74, 74, 74); border-color: rgb(136, 136, 136);"></select></div>
	</div>
	</modelview-dropdown>
```

**Comment by isidorn** [comment]
Those look like elements which are introduced by the Azure data studio extension. So this issue should be fixed on the extension side and it is good you created the issue here.

**Comment by kburtram** [comment]
This should be fixed in an upcoming insiders build and the next monthly release.  Thanks for reporting the bug!

**Comment by leonardder** [comment]
Many thanks for fixing this!
